week1/homework/src/server.js

```diff
@@ -9,7 +9,38 @@ function createServer(port) {
   let state = 10;
 
   const server = http.createServer((request, response) => {
-    // TODO: Write your homework code here
+    let responseJsonObject = {};
+    response.setHeader('Content-Type', 'application/json');
+    switch (request.url) {
+      case '/state': {
+        responseJsonObject.state = state;
+        response.write(JSON.stringify(responseJsonObject));
+        break;
+      }
+      case '/add': {
+        responseJsonObject.state = ++state;
+        response.write(JSON.stringify(responseJsonObject));
+        break;
+      }
+      case '/subtract': {
```

Comment:
This does not break the functionality. But should be `substract` and not `subtract` just a small type :)